app/poros/hourly_forecast.rb

```diff
@@ -0,0 +1,31 @@
+class HourlyForecast
+  def initialize(hourly_weather_data)
+    @hourly_data = hourly_weather_data
+  end
+
+  def next_eight_hours
```

Comment:
Same comment as CurrentWeather and DailyForecast POROs, get rid of this method and the hard coded hash by mapping through the hourly data in the Forecast poro and only passing in a single hour of data when initializing this PORO. Then in initialize you can create instance variables and attr_readers instead of using methods.